src/components/common/searchbar.tsx

```diff
@@ -12,31 +12,16 @@ type Props = {
   setSearchValue: (a: string) => void;
 };
 
-const SearchBar: FC<Props> = ({
-  showSearchButton,
-  setOpenSearchField,
-  openSearchField,
-  searchValue,
-  setSearchValue,
-}) => {
+const SearchBar: FC<Props> = ({ showSearchButton, setOpenSearchField, openSearchField, searchValue, setSearchValue }) => {
+
   return (
     <div className="block">
-      <CustomInputText
-        type="text"
-        className="w-full min-w-[400px] h-10 bg-gray-100  p-4 rounded-full hidden sm:block focus:outline-none placeholder:text-stone-500 placeholder:font-helvetica placeholder:font-medium"
-        placeholder="Search Order #"
-        value={searchValue}
-        onChange={(e) => setSearchValue(e.target.value)}
-      />
-      {showSearchButton && (
-        <Button
-          type="submit"
-          className="-ml-8 sm:hidden"
-          onClick={() => setOpenSearchField(!openSearchField)}
-        >
+      <CustomInputText type="text" className="w-full min-w-[400px] h-10 !bg-gray-100  p-4 !rounded-full hidden sm:block focus:outline-none placeholder:text-stone-500 placeholder:font-helvetica placeholder:font-medium !shadow-none !border-none" placeholder="Search Order #" value={searchValue} onChange={(e) => setSearchValue(e.target.value)} />
```

Comment:
When u save this file, the indentation would have cleaned-up the alignment.
I don't think your alignment changes on saving ?
Does it ?
if not, please fix it. Contact @AdithyaShaj  for this and use her prettier settings.

Reply:
Done